src/features/user/GiftFunds/Details.tsx

```diff
@@ -1,41 +1,77 @@
-import React, { useContext, ReactElement } from 'react';
+import React, { useContext, ReactElement, useState } from 'react';
 import { UserPropsContext } from '../../common/Layout/UserPropsContext';
-import styles from './GiftFunds.module.scss';
 import { useTranslation } from 'next-i18next';
+import { GiftFunds } from '../../common/types/user';
+import { styled } from '@mui/material';
 
 const Details = (): ReactElement | null => {
   const { user } = useContext(UserPropsContext);
   const { t, ready } = useTranslation('giftfunds');
+  const [validGiftFunds, setValidGiftFunds] = useState<GiftFunds[] | null>(
+    null
+  );
+
+  React.useEffect(() => {
+    const nonZeroOpenUnitsGiftFunds = user.planetCash?.giftFunds.filter(
+      (gift) => gift.openUnits !== 0
+    );
+    setValidGiftFunds(
+      nonZeroOpenUnitsGiftFunds ? nonZeroOpenUnitsGiftFunds : null
+    );
+  }, [user]);
+
+  const StyledContainer = styled('div')(({ theme }) => ({
+    backgroundColor: theme.palette.background.default,
+    padding: 34,
+    borderRadius: 9,
+    boxShadow: `0px 2px 1px -1px rgb(0 0 0 / 20%),
+    0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%)`,
+    marginBottom: 24,
+    '& .container_heading': {
+      paddingBottom: 16,
+    },
+    '& .container_details': {
+      display: 'flex',
+      marginTop: 24,
+      flexWrap: 'wrap',
+      gap: 15,
+
+      div: {
+        flex: 1,
+      },
+    },
+    hr: {
+      border: '1px solid #DDDBDA',
+    },
+  }));
```

Comment:
A small change can be made here to ensure that the styles for AccountDetails (PlanetCash), BankAccountDetails (Manage Payouts) and StyledContainer are common. 

Compare the files - 
src/features/user/PlanetCash/components/AccountDetails.tsx
src/features/user/ManagePayouts/components/BankAccountDetails.tsx
src/features/user/GiftFunds/Details.tsx